### ISSUE TITLE
Normalise dc:source lang tags

### ISSUE THREAD

**Issue by paulduchesne** [FIAF/FIAFcore#7]
Currently dc:source tags are language specific when referenced to Cataloguing Manual, but agnostic when taken from ISO resources. This is causing issues with language filtering for mkdocs.

Cataloging Manual sourced:

```
<https://fiafcore.org/ontology/AbbreviatedTitle> a owl:Class ;
    rdfs:label "Abbreviated Title"@en,
        "Título abreviado"@es,
        "Titre abrégé"@fr ;
    dc:source "FIAF Moving Image Cataloguing Manual A.2.4.1"@en,
        "Manual FIAF de Catalogación de Imágenes en Movimiento A.2.4.1"@es,
        "Manuel de catalogage des images animées de la FIAF A.2.4.1"@fr ;
    rdfs:subClassOf <https://fiafcore.org/ontology/AlternativeTitle> .
```

ISO sourced:

```
<https://fiafcore.org/ontology/Abkhazian> a owl:Class ;
    rdfs:label "Abkhazian"@en,
        "Abjasiano"@es,
        "Abkhaze"@fr ;
    dc:source "ISO 639-1"^^xsd:string ;
    rdfs:subClassOf <https://fiafcore.org/ontology/Language> .
```

**Comment by paulduchesne** [comment]
Expansion that all string literals need to have language tags if they are expected to display on the documentation website.

**Comment by paulduchesne** [comment]
Fixed as of 40e82abe4154287a57876bb234a7ad6bd4540504
